postgres/src/test/java/org/jdbi/v3/postgres/TestHStore.java

```diff
@@ -133,6 +134,7 @@ public void testWritesWithBindMap() {
     public interface CampaignDao {
 
         @SqlQuery("select caps from campaigns where id=:id")
+        @SingleValue
```

Comment:
Yep. FWIW we already have to do this when selecting a single row of an array column.